sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -220,19 +218,25 @@ def get_authority(endpoint):
     if ".azurecr.us" in endpoint:
         logger.warning("US Gov Authority:")
         return AzureAuthorityHosts.AZURE_GOVERNMENT
+    if ".azurecr.de" in endpoint:
+        logger.warning("Germany Authority:")
+        return AzureAuthorityHosts.AZURE_GERMANY
     raise ValueError("Endpoint ({}) could not be understood".format(endpoint))
 
 
-def get_authorization_scope(authority):
+def get_audience(authority):
     if authority == AzureAuthorityHosts.AZURE_PUBLIC_CLOUD:
         logger.warning("Public auth scope")
-        return ["https://management.core.windows.net/.default"]
+        return ContainerRegistryAudience.ARM_PUBLIC_CLOUD_VALUE
```

Comment:
It looks like the Python precedent is not to use the `_VALUE` suffix (just looking at `AzureAuthorityHosts`)